core/src/main/scala/org/http4s/sbt/Http4sOrgPlugin.scala

```diff
@@ -91,16 +75,6 @@ object Http4sOrgPlugin extends AutoPlugin {
       githubWorkflowPublishTargetBranches :=
         Seq(RefPredicate.StartsWith(Ref.Tag("v"))),
       githubWorkflowBuild := Seq(
-        WorkflowStep
-          .Sbt(List("scalafmtCheckAll", "scalafmtSbtCheck"), name = Some("Check formatting")),
-        WorkflowStep.Sbt(List("headerCheck", "test:headerCheck"), name = Some("Check headers")),
-        WorkflowStep.Sbt(List("test:compile"), name = Some("Compile")),
-        WorkflowStep.Sbt(List("mimaReportBinaryIssues"), name = Some("Check binary compatibility")),
-        WorkflowStep.Sbt(
-          List("undeclaredCompileDependencies", "unusedCompileDependenciesTest"),
-          name = Some("Check explicit dependencies")),
-        WorkflowStep.Sbt(List("test"), name = Some("Run tests")),
-        WorkflowStep.Sbt(List("doc"), name = Some("Build docs"))
-      )
+        WorkflowStep.Sbt(List("ci")))
```

Comment:
Might regret this.  The seq of steps is easier to modify downstream than the ci alias, and we can quickly see where CI failed in the midst of GitHub Action's decrepit viewer.

Reply:
We could try to do it the other way around: the `ci` command alias is depends on `githubWorkflowBuild`.

Reply:
Oh, that's clever.  I think we'd have to do that upstream, but I like it.

Reply:
Hmm, my suggestion was premature; I have not thought that  `addCommandAlias` can't depend on other tasks for its arguments.

Reply:
Can it with `Def.derive`?

Reply:
I have no experience with `Def.derive`, but I don't know if it is applicable, as `addCommandAlias` returns `Seq[Def.Settings[State => State]]`.

```scala
/**
 * Constructs a derived setting that will be automatically defined in every scope where one of its dependencies
 * is explicitly defined and the where the scope matches `filter`.
 * A setting initialized with dynamic dependencies is only allowed if `allowDynamic` is true.
 * Only the static dependencies are tracked, however.  Dependencies on previous values do not introduce a derived setting either.
 */
final def derive[T](
    s: Setting[T],
    allowDynamic: Boolean = false,
    filter: ScopeType => Boolean = const(true),
    trigger: AttributeKey[_] => Boolean = const(true),
    default: Boolean = false
): Setting[T] = ???
```

Reply:
Oh.  Yeah, that's well over my head.

Instead of an alias, maybe `ci` could be a task?  That seems heavier, though.  We're twisting ourselves in knots here work around sbt-spiewak's `ci`, which might not be as general as that plugin wants it to be.

Reply:
Is there a way to execute tasks by name/`String`? If there is, it should be easy to define `ci` as a task depending on `githubWorkflowBuild`.

Reply:
Um, sbt-release does it, but as `ReleaseStep`s, which I recall are weird.